src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -65,30 +73,48 @@ public static void TestOid_DSA()
         }
 
         [Fact]
+        public static void TestOid_ECDSA()
+        {
+            PublicKey pk = GetTestECDsaKey();
+            Assert.Equal("1.2.840.10045.2.1", pk.Oid.Value);
+        }
+
+        [Fact]
         public static void TestPublicKey_Key_RSA()
         {
             PublicKey pk = GetTestRsaKey();
-            AsymmetricAlgorithm alg = pk.Key;
-            Assert.NotNull(alg);
-            Assert.Same(alg, pk.Key);
-            Assert.Equal(2048, alg.KeySize);
+            using (AsymmetricAlgorithm alg = pk.Key)
+            {
+                Assert.NotNull(alg);
+                Assert.Same(alg, pk.Key);
+                Assert.Equal(2048, alg.KeySize);
 
-            Assert.IsAssignableFrom(typeof(RSA), alg);
-            VerifyKey_RSA(/* cert */ null, (RSA)alg);
+                Assert.IsAssignableFrom(typeof(RSA), alg);
+                VerifyKey_RSA( /* cert */ null, (RSA)alg);
+            }
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Windows)] // Until DSA interop support added
```

Comment:
Looks like this line had a reason to be there after all.  But I don't see an issue, so making one and changing this to [ActiveIssue]